server/plugin/command.go

```diff
@@ -25,6 +26,11 @@ const (
 	featureStars         = "stars"
 )
 
+const (
+	ErrorNoWebhookFound           = "\nNo webhook was found for this repository or organization. To create one, enter the following slash command `/github setup webhook`"
+	GithubListOptionsPerPageValue = 50
```

Comment:
Do we have a `const` for `perPage` anywhere else?

Reply:
no there is no constant for this in the code

Reply:
1/5 let's use a generic `perPage` here and in the rest of the code.